src/server/js/routes/product.js

```diff
@@ -18,14 +18,20 @@
  */
 
 import {Router} from 'express';
+import * as admin from 'firebase-admin';
+
 const router = new Router();
 
-/* GET home page. */
-router.get('/', (req, res, next) => {
-  res.render('product', {
-    title: 'Apple Pie — Pie Shop',
-    product_name: 'Apple Pie',
-    product_description: 'A classic dessert, with our unique exotic spice blend.'});
+/* GET /product/ */
+router.get('/', async (req, res, next) => {
```

Comment:
I've used an `async` on the route here because the server functionality here is pretty much a pass through to Firestore. I tried some other approaches, letting the promise resolve and then returning the results - but this can make things messy if the route returns before the response does. If we add additional middleware later that runs after the route this could bit us. Hence, `async` on the whole thing.